ui/src/containers/CreateVolume.js

```diff
@@ -666,10 +672,34 @@ const CreateVolume = (props) => {
                           name="vgName"
                           value={values.vgName}
                           onChange={handleChange('vgName')}
-                          label={`Volume Group Name*`}
+                          label={`${intl.formatMessage({ id: "vg_name" })}*`}
                           error={touched.vgName && errors.vgName}
                           onBlur={handleOnBlur}
                         />
+                        <Input
+                          name="forceLVCreate"
+                          value={values.forceLVCreate}
+                          onChange={handleChange('forceLVCreate')}
+                          label={
+                            <>
+                              {`${intl.formatMessage({ id: "force_lvcreate" })}`}
+                              <Tooltip
+                                placement="right"
+                                overlay={
+                                  <div style={{ minWidth: '200px' }}>
```

Comment:
nit: the new form component is going to handle this, so you don't have to care about it
```suggestion
<div style={{ minWidth: '80rem' }}>
```